components/screens/TelemetryScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import {createNativeStackNavigator, NativeStackScreenProps} from '@react-navigation/native-stack';
+import {TabNavigatorParamList, TelemetryStackParamList} from 'routes';
+import {StyleSheet, View} from 'react-native';
+import {TelemetryStationData} from 'components/TelemetryStationData';
+import React from 'react';
+import {TelemetryStationMap} from 'components/TelemetryStationMap';
+
+const TelemetryStack = createNativeStackNavigator<TelemetryStackParamList>();
+export const TelemetryTabScreen = ({route}: NativeStackScreenProps<TabNavigatorParamList, 'Weather Data'>) => {
+  const {center_id, date} = route.params;
+  return (
+    <TelemetryStack.Navigator initialRouteName="telemetryStations">
+      <TelemetryStack.Screen
+        name="telemetryStations"
+        component={TelemetryScreen}
+        initialParams={{center_id: center_id, date: date}}
+        options={() => ({title: `${center_id} Telemetry Stations`})}
+      />
+      <TelemetryStack.Screen
+        name="telemetryStation"
```

Comment:
this shows up as a transparent card right now, only for the first random stations, is the idea to have the stations be clickable kind of like avalanche forecast zone cards? honestly might just be easier to have the clickable part be the name that appears when the pin is clicked?

Reply:
Yeah, I was expecting to make these two UX bits similar once we have them figured out, just slapped something trivial together in the past when I created the telemetry screen to prove out the SnowBound API - I think having the pin clickable would be good, or having it auto-scroll to the correct card (and show some metadata about what sensors exist) would also be good.